package.json

```diff
@@ -312,91 +312,9 @@
-		"@wordpress/block-serialization-spec-parser": "4.49.0",
-		"@wordpress/blocks": "^12.26.0",
-		"@wordpress/browserslist-config": "5.32.0",
 		"@wordpress/components": "^25.15.0",
-		"@wordpress/compose": "6.26.0",
-		"@wordpress/core-data": "6.26.0",
-		"@wordpress/create-block-tutorial-template": "3.3.0",
-		"@wordpress/create-block": "4.33.0",
-		"@wordpress/custom-templated-path-webpack-plugin": "2.13.0",
-		"@wordpress/customize-widgets": "4.26.0",
-		"@wordpress/data-controls": "3.18.0",
-		"@wordpress/data": "^9.19.0",
-		"@wordpress/date": "4.49.0",
-		"@wordpress/dependency-extraction-webpack-plugin": "5.0.0",
-		"@wordpress/deprecated": "3.49.0",
-		"@wordpress/docgen": "1.58.0",
-		"@wordpress/dom-ready": "3.49.0",
-		"@wordpress/dom": "3.49.0",
-		"@wordpress/e2e-test-utils": "10.20.0",
-		"@wordpress/e2e-tests": "7.20.0",
-		"@wordpress/edit-post": "7.26.0",
-		"@wordpress/edit-site": "5.26.0",
-		"@wordpress/edit-widgets": "5.26.0",
-		"@wordpress/editor": "13.26.0",
 		"@wordpress/element": "^5.26.0",
-		"@wordpress/escape-html": "2.49.0",
-		"@wordpress/eslint-plugin": "^17.6.0",
-		"@wordpress/experiments": "0.8.0",
-		"@wordpress/format-library": "^4.26.0",
-		"@wordpress/hooks": "3.49.0",
-		"@wordpress/html-entities": "3.49.0",
-		"@wordpress/i18n": "^4.49.0",
-		"@wordpress/icons": "^9.40.0",
-		"@wordpress/interface": "5.26.0",
-		"@wordpress/is-shallow-equal": "^4.49.0",
-		"@wordpress/jest-console": "7.20.0",
-		"@wordpress/jest-preset-default": "11.20.0",
-		"@wordpress/jest-puppeteer-axe": "6.20.0",
-		"@wordpress/keyboard-shortcuts": "4.26.0",
-		"@wordpress/keycodes": "^3.49.0",
-		"@wordpress/lazy-import": "1.36.0",
-		"@wordpress/library-export-default-webpack-plugin": "2.14.0",
-		"@wordpress/list-reusable-blocks": "4.26.0",
-		"@wordpress/media-utils": "4.40.0",
-		"@wordpress/notices": "4.17.0",
-		"@wordpress/npm-package-json-lint-config": "4.34.0",
-		"@wordpress/nux": "8.11.0",
-		"@wordpress/plugins": "6.17.0",
-		"@wordpress/postcss-plugins-preset": "4.33.0",
-		"@wordpress/postcss-themes": "5.32.0",
-		"@wordpress/preferences-persistence": "1.41.0",
-		"@wordpress/preferences": "3.26.0",
-		"@wordpress/prettier-config": "3.6.0",
-		"@wordpress/primitives": "^3.47.0",
-		"@wordpress/priority-queue": "2.49.0",
-		"@wordpress/private-apis": "^0.31.0",
-		"@wordpress/project-management-automation": "1.48.0",
-		"@wordpress/react-i18n": "3.47.0",
-		"@wordpress/readable-js-assets-webpack-plugin": "2.32.0",
-		"@wordpress/redux-routine": "4.49.0",
-		"@wordpress/reusable-blocks": "4.26.0",
-		"@wordpress/rich-text": "^6.26.0",
-		"@wordpress/server-side-render": "4.26.0",
-		"@wordpress/shortcode": "3.49.0",
-		"@wordpress/style-engine": "1.32.0",
-		"@wordpress/stylelint-config": "^21.32.0",
-		"@wordpress/token-list": "2.49.0",
-		"@wordpress/url": "^3.50.0",
-		"@wordpress/viewport": "5.26.0",
-		"@wordpress/warning": "2.49.0",
-		"@wordpress/widgets": "3.26.0",
-		"@wordpress/wordcount": "3.49.0"
```

Comment:
The reason I liked having them is because it's pretty easy for someone to add a different version of a package in a package somewhere in the monorepo, and this ensures everyone is on the supported version. You're right, it's not as necessary now, but I liked the extra safety. Just my 2c :)

Reply:
Makes sense, and definitely doesn't hurt keeping that. Thanks for providing your feedback - going to close.